src/server/saved_objects/client/lib/parse_es_doc.js

```diff
@@ -0,0 +1,16 @@
+import { get } from 'lodash';
+
+export function parseEsDoc(doc, overrides = {}) {
```

Comment:
If we know the doc's type, we could potentially simplify this logic by retrieving the different fields from the different "paths" if it matches the new v6 "doc" type